store/wrapper_txn.go

```diff
@@ -4,6 +4,16 @@ import (
 	"bytes"
 	"github.com/canopy-network/canopy/lib"
 	"github.com/dgraph-io/badger/v4"
+	"reflect"
+	"unsafe"
+)
+
+const (
+	badgerMetaFieldName               = "meta"    // badgerDB Entry 'meta' field name
+	badgerDiscardEarlierVersions byte = 1 << 2    // badgerDB 'discard earlier versions' flag
```

Comment:
although this codebase is not a guide on how to implement managed badgerdb, linking the [source](https://github.com/hypermodeinc/badger/blob/main/value.go) of these bits will help future readers/maintainers of the codebase as this is now a simple yet integral piece of how the store works